### PR TITLE
Resurrection time & styled pagination dropdown

### LINKED ISSUE #187: Edit resurrection time + styling pagination dropdown
Notes

- edit/hide button when selecting radio (copy Resurrection component)

### PR DESCRIPTION
**Pull request by solidoracle**
### Overview
In the Select Arch page in the Embalm process I added the possibility to edit the resurrection time (linked to global state)

I also styled the Select component of the pagination on the bottom left of the page
Screenshot 2022-11-17 at 09 47 19

### CODE REVIEW

File: src/features/embalm/stepContent/components/SetPaginationSize.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            fontSize: '14px',
+            borderColor: 'violet.700',
+          }),
+
+          option: (provided, state) => ({
+            ...provided,
+            background: state.isFocused ? 'brand.100' : provided.background,
+            fontSize: '14px',
+            color: 'brand.600',
+            my: '-0.2rem',
```
Comment (knoll3) [inline src/features/embalm/stepContent/components/SetPaginationSize.tsx:47]: This is super picky but could you just add this for when the option is clicked please? 
```
  _active: {
    bg: 'brand.200',
  },
```

File: src/features/embalm/stepContent/components/SetPaginationSize.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import { Box } from '@chakra-ui/react';
+import { Select, OptionBase, GroupBase } from 'chakra-react-select';
+
+export interface IPageSizeSetByOption extends OptionBase {
+  value: number;
+  label: string;
+}
+
+export function SetPaginationSize({ handlePageSizeChange, paginationSize }: any) {
+  const PageSizeOptionsMap: IPageSizeSetByOption[] = [
```
Comment (knoll3) [inline src/features/embalm/stepContent/components/SetPaginationSize.tsx]: Could you rename this to `pageSizeOptionsMap` with a lower case please?

File: src/features/embalm/stepContent/steps/SelectArchaeologists.tsx [hunk 3]
```diff
@@ -59,24 +61,61 @@ export function SelectArchaeologists() {
...
+            }}
+          >
+            ({resurrectionTimeEdit ? 'hide' : 'edit'})
+          </Text>
+        </>
+      </HStack>
+      {resurrectionTimeEdit && (
+        <SetResurrection
+          mt={5}
+          w={'50%'}
```
Comment (knoll3) [inline src/features/embalm/stepContent/steps/SelectArchaeologists.tsx]: I think that the width here in this case would be better as a fixed width for now. At 50% the radio labels wrap at certain screen sizes. Maybe something like `w='500px'`.

**Review by knoll3 — APPROVED** [review]
Nice work. After addressing the minor styling comments feel free to merge.

**Comment by solidoracle** [comment]
> Nice work. After addressing the minor styling comments feel free to merge.

Thanks! Addressed the comments and merging now

**Pull request merged**